base/managers.jl

```diff
@@ -79,6 +79,9 @@ Keyword arguments:
 * `dir`: specifies the working directory on the workers. Defaults to the host's current
          directory (as found by `pwd()`)
 
+ * `enable_threaded_blas`: if `true` then  BLAS will run on multiple threads in added
+                           processes. Default is `false`.
+
```

Comment:
Can you add a line here https://github.com/JuliaLang/julia/blob/6b6dead59db79f7940ad2d54d5269e5a319ed796/base/managers.jl#L302 stating that keyword args `dir`, `exename`, `exeflags`, `topology` and `enable_threaded_blas` have the same effect as documented for `addprocs(machines)`